state/runtime/evm/instructions_test.go

```diff
@@ -860,21 +872,25 @@ func TestCallValue(t *testing.T) {
 }
 
 func TestCallDataLoad(t *testing.T) {
-	t.Run("Zero", func(t *testing.T) {
+	t.Run("NonZeroOffset", func(t *testing.T) {
 		s, cancelFn := getState(&chain.ForksInTime{})
 		defer cancelFn()
 
 		s.push(one)
 
+		s.msg = &runtime.Contract{Input: big.NewInt(7).Bytes()}
```

Comment:
How this was working before and why do you need it now? Asking just from curiosity.